lib/rspec/support/spec/library_wide_checks.rb

```diff
@@ -90,8 +88,8 @@ def have_successful_no_warnings_output
   end
 
   it 'only loads a known set of stdlibs so gem authors are forced ' \
-     'to load libs they use to have passing specs', :slow, :failing_on_appveyor do
-    loaded_features = File.read(loaded_features_outfile).split("\n")
+     'to load libs they use to have passing specs', :slow do
+    loaded_features = @loaded_features.split("\n")
```

Comment:
It's a little weird to use "loaded_features" for the name of both variables when they are different (array vs string).  Maybe make the instance variable be `@loaded_features_lines` to make the difference more obvious?